### DISCUSSION TITLE
Finally switching to Vuetify

### DISCUSSION THREAD

**Discussion by 0x00A1FA** [quasarframework/quasar#16628] [Show and Tell]
Hi,

Thank you so much for making it so apparent that Quasar is dead. I can now safely and surely switch to Vuetify.

Thank you!

**Comment by vrilcode** [comment]
What happened?

**Comment by rstoenescu** [comment]
This is nonsense. When you'll see what he have been working on under the covers you'll come to the opposite conclusion by yourself.

**Comment by metalsadman** [comment]
this is not an airport lol
